tests/transchunk.test/t1.sql

```diff
@@ -0,0 +1,32 @@
+-- DRQS-172914411
+
+drop table if exists t1;
+
+create table t1 {
+    schema {
+        int c1
+        int c2
+    }
+    keys {
+        "key" = c1 + c2 {where c1 = 1}
+    }
+}$$
+
+set transaction blocksql
+set transaction chunk 1
+begin;
+delete from t1;
+insert into t1 values (1, 1);
+insert into t1 values (1, 1);
+commit;
+
+-- Since the above transaction ran in chunk-mode, only the first INSERT
+-- should have succeeded.
+select count(*) = 1 from t1;
```

Comment:
I don't understand this outcome. Why won't both inserts work?

Reply:
The other record will fail due to duplicate key error as it will try to commit at part of another (2nd chunked-)transaction.

Reply:
Ah! Got it.